### PR TITLE
Christian/c 7679 hosted page styling fixes

### PR DESCRIPTION
**Pull request by heyglassy**
## Description

Style polish so that preference page matches [Figma](https://www.figma.com/file/o5xrU5ry2GotmymjOhBonX/Preferences-V4.0?node-id=1836%3A11888&t=m3uF5hHxujy3ez0K-0) more closely.

image

https://www.loom.com/share/18437aad4f944811b6e88aba53250ac4

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/components/src/components/PreferencePage.tsx [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
   draft = false,
 }) => {
   return (
-    <div>
-      <Header draft={draft} />
-      <PreferenceList draft={draft} />
-      <Footer draft={draft} />
+    <div
+      style={{
+        height: "100vh",
```
Comment (joshshowalter) [inline packages/components/src/components/PreferencePage.tsx:10]: any reason for doing inline-styles here insteads of classes?

**Review by misabel — DISMISSED** [review]
lgtm, pls post loom summarizing changes

File: packages/components/src/components/PreferencePage.tsx (hunk 1 above)
Comment (heyglassy) [reply packages/components/src/components/PreferencePage.tsx]: moved it over to styled components!

**Comment by heyglassy** [comment]
> lgtm, pls post loom summarizing changes

Here is the loom! https://www.loom.com/share/18437aad4f944811b6e88aba53250ac4

**Review by misabel — APPROVED** [review]
lgtm!

**Pull request merged**